libs/MeshKernelApi/tests/src/Mesh2DRefinmentTests.cpp

```diff
@@ -566,7 +566,9 @@ class MeshRefinementSampleValueTypes : public ::testing::TestWithParam<meshkerne
     [[nodiscard]] static std::vector<meshkernel::InterpolationValues> GetData()
     {
         return {meshkernel::InterpolationValues::shortType,
```

Comment:
- Rename function to GetDataTypes
- reorder if you move int above double in the enum, even though it does not matter (you set the values of the enums)